packages/web/docs/src/pages/docs/gateway/other-features/upstream-reliability.mdx

```diff
@@ -0,0 +1,212 @@
+---
+description: Improve reliability of Hive Gateway with timeout and retry policy for upstream requests
+---
+
+# Upstream reliability
```

Comment:
![Screenshot 2024-12-17 at 09 20 06](https://github.com/user-attachments/assets/f4730829-7ea8-4f15-a025-b7bb81ca74df)

Maybe instead of:
```
Retry
  Configuration
    Configuration by subgraph
    Delay
    Backoff
    Skip a request
Timeout
  Configuration
    Configuration by subgraph
Mixing Retry and Timeout
```

Maybe we should do:

```
Retry mechanism
  Global configuration
  Per-subgraph configuration
  Retry delay configuration
  Exponential backoff settings
  Conditional retry logic
Timeout management
  Global configuration
  Per-subgraph configuration
Combining retry and timeout features
```

@ardatan @EmrysMyrddin @dotansimha 

What do you think?

Reply:
Thank you ! I was struggling very hard to find proper titles here ^^'